packages/nvidia_nat_langchain/tests/eval/test_langsmith_judge.py

```
@@ -0,0 +1,545 @@
+                "inputs": "Q2", "outputs": "A2", "score": 0.5, "reasoning": "Partial"
+            },
+        ]
+        config = LangSmithJudgeConfig(
+            prompt="correctness",
+            llm_name="eval_llm",
+            few_shot_examples=examples,
+        )
+        assert len(config.few_shot_examples) == 2
+        assert config.few_shot_examples[0]["score"] is True
+
+    async def test_system_passed_to_create_llm_as_judge(self, eval_input_matching):
```

Ruff (0.15.0)

[warning] Unused method argument: `eval_input_matching`

(ARG002)